reports/graphs/breastpump_amounts.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+from django.utils import timezone
+from django.utils.translation import gettext as _
+
+import plotly.offline as plotly
+import plotly.graph_objs as go
+
+from reports import utils
+
+
+def breastpump_amounts(objects):
+    """
+    Create a graph showing breastpump amounts over time.
+    :param instances: a QuerySet of Breastpump instances.
+    :returns: a tuple of the the graph's html and javascript.
+    """
+    objects = objects.order_by("-time")
+
+    trace = go.Scatter(
```

Comment:
I feel like this would be more useful as a bar graph, no?

Reply:
I'm not sure. Do you think you could like doodle up what you're thinking for a graph? I'm a little unclear on it. I'd also consider just starting with a simple bar graph and then working out a more advanced one in a new issue.

Reply:
The best example I could think of was through my Grafana instance. Here's a screenshot of some metrics that I have. You're able to set 'intervals' on Grafana. The top screenshot is over a 1 day interval and the bottom screenshot is the same graph, but over a 5 minute interval. Basically what the wife was asking is if it would be able to toggle between a "total 1 day amount" and "amounts per entry" type of graph... So potentially similar to these Grafana graphs (but bar graphs would probably be prettier than line graphs). My guess is that it'd make more sense to have 2 different graph dashboards to do this, but if it's possible to do with the existing module, that'd be awesome (just some toggle switch or something for intervals?)

![intervals](https://user-images.githubusercontent.com/88092414/156800174-773b9463-863f-480e-8d18-ce444caf04bb.png)

Reply:
Maybe take a look at the Feeding Amounts by Type graph? It uses stacked bar graphs so you can stack the entries and sum them by day to get pretty much the effect you're going for I think.

Reply:
Ah hah, thanks for pointing that one out! I didn't look enough into that. Asked the wife and she says yes so I'll go ahead and try that out! :)

Reply:
Haha. Yeah Plotly is kind of intense. Honestly I've always felt like it's overkill as part of Baby Buddy. We can always shift this off to another issue if you get stuck.